jbmc/src/java_bytecode/select_pointer_type.h

```diff
@@ -47,6 +57,24 @@ class select_pointer_typet
     const irep_idt &parameter_name,
     const namespacet &ns) const;
 
+  /// Specialize generic parameters in a pointer type based on the current map
```

Comment:
:pick: "in a pointer type" -> "in a pointer type representing a generic parameter or an array of generic parameters, ..."? It looks like those are the only two cases where this function does anything?

Reply:
I updated the doc a bit to make it more clear